web/api/queries/median/README.md

```diff
@@ -1,8 +1,7 @@
 <!--
----
 title: "Median"
+description: "Use median in API queries and health entities to find the "middle" value from a sample, eliminating any unwanted spikes in the returned metrics."
```

Comment:
you've got double quotes inside double quotes; not sure if that's a problem per earlier issues